log.js

```diff
@@ -1,17 +1,11 @@
-const winston = require('winston')
-const { AzureApplicationInsightsLogger } = require('winston-azure-application-insights')
+// eslint-disable-next-line import/order
 const appInsights = require('./azure-appinsights')
+const { AzureApplicationInsightsLogger } = require('winston-azure-application-insights')
```

Comment:
ensuring app insights is registered first so it can do it's magical instrumentation on winston